### PR TITLE
Fix styling in Safari

### PR DESCRIPTION
**Pull request by pyrmont**
This improves the styling in Safari, namely:

- the index page properly aligns the previews in the second column; and
- the lexer page uses the negative margin trick to get rid of the clearfix hack.

### CODE REVIEW

**Comment by pyrmont** [comment]
Strictly speaking, the clearfix hack isn't a problem in Safari but it irritated me to have non-semantic HTML containers in the code :P

File: spec/visual/templates/index.erb [hunk 1]
```diff
@@ -1,17 +1,19 @@
 <style type="text/css">
   main {
-    padding: 15px;
     column-count: 2;
     -webkit-column-count: 2;
     -moz-column-count: 2;
+    padding: 15px;
```
Comment (ashmaroli) [inline spec/visual/templates/index.erb:6]: why this change..?

File: spec/visual/templates/index.erb [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
-    margin-bottom: 15px;
-    padding: 15px;
-    color: <%= @comment_color %>;
     border: 1px dotted;
     break-inside: avoid-column;
     -webkit-column-break-inside: avoid;
+    color: <%= @comment_color %>;
+    display: inline-block;
+    margin-bottom: 15px;
+    padding: 15px;
```
Comment (ashmaroli) [inline spec/visual/templates/index.erb:15]: Why make these changes..?

File: spec/visual/templates/index.erb (hunk 1 above)
Comment (pyrmont) [reply spec/visual/templates/index.erb:6]: You don't actually need to accept this one. I just like alphabetically ordering my CSS rules to help me with editing.

File: spec/visual/templates/index.erb (hunk 2 above)
Comment (pyrmont) [reply spec/visual/templates/index.erb:15]: The alphabetical ordering changes are for the same reason as the previous comment. The reason to use `display: inline-block` is that it ensures that the first `.preview` item in the second column is placed at the very top of the column in Safari. Not doing this causes the item to be slightly lower than the item in the first column.

Comment (ashmaroli) [reply spec/visual/templates/index.erb:15]: I don't have issues with `display: block`..
Just that I'm used to ordering attributes in the fashion:
```
position
float
display
margin
padding
font
background
border
```

Comment (pyrmont) [reply spec/visual/templates/index.erb:15]: That's fine. The other rules aren't alphabetically ordered so my change is hardly consistent. I just personally find it easier to do alphabetical ordering than remember an arbitrary list of groupings.

**Comment by ashmaroli** [comment]
Rouge doesn't have a css-linter. So it doesn't matter :man_shrugging:
